frontend/express/public/core/app-management/javascripts/countly.views.js

```diff
@@ -11,7 +11,7 @@
                 set: function(value) {
                     this.newApp = false;
                     this.selectedApp = value;
-                    this.broadcast('AppSettingsContainerObservable', 'selectedApp', value);
```

Comment:
when a new application is created, it re-renders the plugins collapse view. As a consequence, no view that is subscribed to "selectedApp" event will not pick it up. Ultimately, this leads to stale data in those views as they still think that the active app id is the one before the new app was created.